<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TreestaImporterDialog</class>
 <widget class="QDialog" name="TreestaImporterDialog">
  <property name="windowTitle">
   <string>Treesta Importer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="inputLayout">
     <item>
      <widget class="QLineEdit" name="lineEditInput">
       <property name="placeholderText">
        <string>Pfad zur alten CSV-Dateiâ€¦</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBrowse">
       <property name="text">
        <string>Durchsuchenâ€¦</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnConvert">
     <property name="text">
      <string>Umwandlung starten</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelStatus">
     <property name="text">
      <string>Statusmeldungâ€¦</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditUnmapped">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Nicht gemappte Werte werden hier angezeigtâ€¦</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnOpenFolder">
     <property name="text">
      <string>ðŸ“‚ Ausgabeordner Ã¶ffnen</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
